mlir-tensorrt/tensorrt/test/TensorRT/dynamic-shape-inference.mlir

```diff
@@ -0,0 +1,45 @@
+// tensorrt-opt %s -split-input-file -test-tensorrt-shape-inference | FileCheck %s
+
+func.func @test_resize_linear(%arg0: tensor<10x10xf32>) -> (index, index) {
+  %result = tensorrt.resize_linear {
+    coordinateTransformation = #tensorrt.resize_coordinate_transformation<kALIGN_CORNERS>,
+    selectorForSinglePixel = #tensorrt.resize_selector<kUPPER>
+  } %arg0 : (tensor<10x10xf32>) -> tensor<20x20xf32>
+
+  %c0 = arith.constant 0 : index
+  %c1 = arith.constant 1 : index
+  %d0 = tensor.dim %result, %c0 : tensor<20x20xf32>
+  %d1 = tensor.dim %result, %c1 : tensor<20x20xf32>
+  return %d0, %d1 : index, index
+}
+
+// CHECK-LABEL: test_resize_linear
+// CHECK-NEXT: %[[c20:.+]] = arith.constant 20 : index
+// CHECK-NEXT: return %[[c20]], %[[[c20]]] : index, index
+
+// -----
+
+func.func @test_resize_dynamic_batch(%arg0: tensor<?x1x10x10xf32>) -> (index, index, index, index) {
+  %result = tensorrt.resize_linear {
+    coordinateTransformation = #tensorrt.resize_coordinate_transformation<kALIGN_CORNERS>,
```

Comment:
You'll need to add a test where `output_shape` is used. These two tests only cover static result shapes (in spatial dimensions)

Reply:
Added.